scripts/talos/README.md

```diff
@@ -0,0 +1,61 @@
+# Talos
+
+## Description
+
+Talos is a bumper bot which runs automatically on the latest branch of Psammead. Talos was created to make life easier for developers, with Talos they do not need to keep track of their dependencies, it saves time, makes sure dependencies are up to date and it will prevent a duplicate package bundle from existing in external applications when multiple Psammead packages are used. Furthermore, it bumps everything packages in 2 PRs, therefore it is quicker to review PRs, whereas Dependabot would do the same in more than 2 PRs.
+
+Simorgh uses `inspectpack` which checks if there are any duplicate package bundles in our application, if packages are duplicated they will not be accepted and will be flagged.
+
+Talos solves this issue as it ensure the package version is the same across the whole application (Psammead).
+
+## Manual Usage
+
+Manual usage of Talos requires access to our Jenkins CI pipeline. So for now it is limited to those employed by the BBC. If you're an external contributor and want Talos run manually, please feel free to [open an issue](https://github.com/bbc/psammead/issues/new/choose). 
+
+### Using Jenkins
+
+Navigate to [Jenkins](https://ci.news.tools.bbc.co.uk/blue/organizations/jenkins/psammead/branches) and click on the play (run) button, then enter the list of packages you want to bump seperated by commas.
+
+<img align="right" alt="screenshot showing the Jenkins interface, starting a Talos job updating react,styled-components" src="https://user-images.githubusercontent.com/34196381/63758525-d03ef980-c8b3-11e9-9b8c-d4f9a451237b.png">
+
+### Using CLI (not recommended)
+
+Talos uses the Github API, therefore requires a token. Talos can be used if you setup the Github token, but its is not recommended.
+
+## FAQs
+
+#### How to review PRs?
+
+PRs can be reviewed like any other PR. The PRs should be tested according to what is being bumped.
+
+#### When a Talos PR gets conflicts how can I rebase/recreate it?
+
+Manually invoking Talos and entering the packages you want to bump can resolve conflicts.
+
+#### Does Talos auto-update itself with the latest branch?
+
+Talos only runs on latest therefore does not need to.
+
+#### Does Talos handle the package bumping and changelog?
+
+Yes. When you make a Psammead package change you publish a new version of it.
+
+When these changes are published, Talos will find where the packages are used across Psammead and will bump them, increasing the version number, then writes a changelog, makes a branch and creates a pull request.
+
+Developers and testers need to review, test and merge the PR.
```

Comment:
We chould answer the question: `how do we test Talos PRs?`

@PriyaKR may have a good answer to this.

Reply:
how do we test Talos PRs?

If the PR just updates the dev dependencies there wouldn't be any testing required from the test team.But if there are other dependencies updated then a regression testing along with testing of the changes gone in that version for the individual component is required.